### PR TITLE
minor changes in prequal error variables

### PR DESCRIPTION
**Pull request by archilsharashenidzebroad**

### CODE REVIEW

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java [hunk 1]
```diff
@@ -146,26 +146,19 @@ private void changeQuetionStyle(Handle handle, long activityId, String stableId)
...
         helper.updateTemplateText(templateText, templateId);
 
-        long questionStableCodeId2 = helper.getQuestionStableCodeId(stableId2);
-        long questionId2 = helper.getQuestionId(questionStableCodeId2).get(0);
-        long validationId2 = helper.getValidationId(questionId2).get(0);
-        helper.insertUpperRange(age, validationId2);
-        Long templateId2 = helper.getTemplateVariableId(varNameChild).get(0);
+        long templateId2 = helper.getTemplateVariableId(varNameChild).get(0);
         helper.updateTemplateText(templateText, templateId2);
     }
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java:172]: Updating age restriction from 100 to 110 and error text

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java [hunk 2]
```diff
@@ -186,21 +179,21 @@ private void changeAgeRestriction(SqlHelper helper) {
...
+                + "order by question_prompt_template_id asc")
         List<Long> getQuestionPromptId(@Bind("stableId") long stableId);
 
         @SqlQuery("select template_variable_id from template_variable where variable_name = :name order by template_variable_id desc")
         List<Long> getTemplateVariableId(@Bind("name") String name);
 
         @SqlQuery("select template_variable_id from template_variable where template_id = :templateId and variable_name = :name "
-                + "order by template_variable_id desc")
+                + "order by template_variable_id asc")
         List<Long> getTemplateVariableIdbyTemplateId(@Bind("templateId") long templateId, @Bind("name") String name);
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java:190]: retrieving data of lowest ids for template

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java [hunk 3]
```diff
@@ -186,21 +179,21 @@ private void changeAgeRestriction(SqlHelper helper) {
...
 
         @SqlQuery("select question_id from question where question_stable_code_id = :stableId order by question_id desc")
         List<Long> getQuestionId(@Bind("stableId") long stableId);
 
-        @SqlQuery("select validation_id from question__validation where question_id = :questionId order by validation_id asc")
-        List<Long> getValidationId(@Bind("questionId")long questionId);
+        @SqlQuery("select validation_id from int_range_validation where max = :max order by validation_id asc")
+        List<Long> getValidationId(@Bind("max")int max);
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java]: updating query from selecting via id to value selection

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java [hunk 4]
```diff
@@ -110,28 +110,28 @@ private void updateQuestion(Handle handle, Config dataCfg, long activityId) {
...
             for (Config config1 : question) {
                 String varName = config1.getString("varName");
                 String subsValue = config1.getString("newVal");
-                long questionId = helper.getQuestionStableCodeId(stableId);
-                long questionPromptId = helper.getQuestionPromptId(questionId).get(0);
-                long templateVariableIdbyTemplateId = helper.getTemplateVariableIdbyTemplateId(questionPromptId, varName).get(0);
+                long templateVariableIdbyTemplateId = helper.getTemplateVariableIdbyTemplateId(questionDto.getPromptTemplateId(), varName);
                 helper.updateTemplateText(subsValue, templateVariableIdbyTemplateId);
             }
         });
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java:132]: For each of the questions, update variables and validations

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java [hunk 5]
```diff
@@ -144,7 +144,7 @@ private void changeQuetionStyle(Handle handle, long activityId, String stableId)
         helper.updatePicklistOption(questionDto.getId(), pickListModeIdByValue);
```
Comment (archilsharashenidzebroad) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoPrequalUpdate.java:144]: Update picklist question rule type from single to multiple

**Review by nsakande — APPROVED** [review]
LGTM

**Pull request merged**